src/sentry/api/serializers/models/release.py

```diff
@@ -107,11 +108,18 @@ def _get_commit_metadata(self, item_list, user):
             # Increment commit count per release
             commit_count_by_release_id[rc.release_id] += 1
 
+            # look for latest commit by release
+            if latest_commit_by_release_id.get(rc.release_id) is None \
+                    or latest_commit_by_release_id.get(rc.release_id).order < rc.order:
+                latest_commit_by_release_id[rc.release_id] = rc
```

Comment:
A "higher" `order` value means it comes after? e.g. `order=0` is the *first* commit?